specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2022-11-01-preview/pds.json

```diff
@@ -0,0 +1,463 @@
+      ]
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "description": "Represents the error response model.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "PolicyElement": {
+      "type": "object",
+      "description": "Represents a policy element.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Id of the policy element."
+        },
+        "scopes": {
+          "type": "array",
+          "description": "List of scopes included in this policy.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "type": "string",
+          "description": "Kind of policy element. Possible values: policy/policySet/attributeRule."
+        },
+        "updatedAt": {
+          "type": "string",
+          "description": "Time at which this policy element is last updated."
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Current version of policy element. This is auto incremented with each change. Starts with 1."
+        },
+        "elementJson": {
+          "type": "string",
+          "description": "Policy element serialized json payload."
+        }
+      }
+    },
+    "PolicyElementEventModel": {
+      "type": "object",
+      "description": "Represents a policy element event model.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Id of the policy element."
+        },
+        "scopes": {
+          "type": "array",
+          "description": "List of scopes included in this policy.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "type": "string",
+          "description": "Kind of policy element. Possible values: policy/policySet/attributeRule."
+        },
+        "updatedAt": {
+          "type": "string",
+          "description": "Time at which this policy element is last updated."
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer",
+          "description": "Current version of policy element. This is auto incremented with each change. Starts with 1."
+        },
+        "elementJson": {
+          "type": "string",
+          "description": "Policy element serialized json payload."
+        },
+        "eventType": {
+          "type": "string",
+          "description": "Type of event. Format: <RP-Name>/<Resource-Type>/<Action>. For Example: Microsoft.Purview/PolicyElements/Delete."
+        }
+      }
+    },
+    "PolicyEventsResponse": {
+      "type": "object",
+      "description": "Represents the response for policy events API.",
+      "properties": {
+        "count": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Count of elements in payload."
+        },
```

Comment:
Don't return count. See my detailed comment below.

Reply:
Hi Mike, we will do this change in the GA API.
But I am afraid that we cannot accommodate this change in this Preview version.
Resolving this comment for now.